pairwise-sentences.R

```diff
@@ -0,0 +1,41 @@
+
+target <- chars[[which(names(chars) =="#Hamlet_Ham")]]
+source <- chars[[which(names(chars) == "#Ophelia_Ham")]]
+
+all_sim <- list() #list containing all cosine similarity between target sentence and previous source sentences
+max_sim <- list() #list containing maximum cosine similarity between a target sentence and previous source sentences
+pairsim_sentnces <- list()
+
+for(j in 1:nrow(target)){
+for(i in 1:nrow(source)){
+  if(source[i,2]<target[j,2]){
+    all_sim[[i]] <-cosine(a1[[as.numeric(target[j,2])]], 

```

Comment:
If there's a reproducibility goal here, this could be written as lsa::cosine or `library(cosine)` run above this in the file -- it is not great practice to have a function in a file that is not imported in the file. same with `library(dplyr)`